src/components/slds-tooltip/index.vue

```diff
@@ -1,7 +1,7 @@
 <template>
     <div class="slds-tooltip__container">
 
-        <slot name="trigger"/>
+        <slot name="trigger" @click="show"/>
```

Comment:
Para eventos eu estou usando o seguinte padrão para nomear os métodos:

`on[Event][Target]`

ou seja, nesse caso para seguirmos o padrão ficaria **onClickTrigger**.

Eu percebi que isso é bem útil não somente pra leitura, mas no futuro se você for reutilizar a lógica de show customizando sua chamada você poderia ter apenas um método show impendente dessa sua trigger. Por exemplo:

```
onClickTrigger() {
    this.show();
},

onClickRefresh() {
    this.doSomething();
    this.show();
},

show() {
    this.isVisible = true;
},
```

Reply:
E  nome do evento que eu emito?

Reply:
Por sinal, se quando você encontrar algum método que lida com eventos fora desse padrão (você eventualmente vai encontrar), se puder adequá-los, fico grato!

Reply:
> E nome do evento que eu emito?

O nome do seu alvo mesmo, por exemplo, vamos supor que eu tenha um botão de Save e um de Delete, eu teria os seguintes métodos:

- onclickSave
- onClickDelete

Se não houver mais nada clicável, digamos o seu componente é um botão, eu nomearia apenas como onClick, porque e;e é o principal e não terá outro no mesmo componente.

Reply:
O tooltip nao era pra estar aqui. Vou remover porque ta em WIP ainda. 
Eu tenho 2 botoes. Close e Back. Vou mudar os handlers the onCloseButton e onBackButton. Mas eles emitem eventos. Posso deixar close e back mesmo? Porque fica mais facil de usar


```html
          <slds-panel
                ref="panel"
                left
                back-button
                small
                title="Panel"
                @close="doSomethingOnClose">
                Panel with Back Button
            </slds-panel>
          <slds-panel
                ref="panel"
                left
                back-button
                small
                title="Panel"
                @back="doSomethingOnBackButton">
                Panel with Back Button
            </slds-panel>
````

Reply:
Eu removi as mudancas do tooltip dessa branch.

Reply:
Sim, quanto à emissão de eventos pro parent nomeia eles do jeito que faça mais sentido, do jeito que você exemplificou está perfeito.

Sobre o parent, seguindo a mesma lógica, nesse seu exemplo ai, os métodos do parent ficariam:

- onBack
- onclose

Porque esses são os nomes dos eventos, e não tem mais nada que parecido com eles. Obviamente isso falando do vuetning né, quem for utilizar vai dar o nome que bem entender, mas dentro da lib seria bom manter um padrão.